lib/gui/pages/settings/settings.js

```diff
@@ -25,7 +25,8 @@ const MODULE_NAME = 'Etcher.Pages.Settings';
 const SettingsPage = angular.module(MODULE_NAME, [
   require('angular-ui-router'),
   require('../../components/warning-modal/warning-modal'),
-  require('../../models/settings')
+  require('../../models/settings'),
+  require('../../modules/error')
```

Comment:
Coincidence. Another module probably required the error module before this module started, so it all just worked. The Angular module system is awful though, and doesn't give any warning for these kind of things. React should make things better.